src/fu-device.c

```diff
@@ -1554,6 +1559,47 @@ fu_device_set_remove_delay (FuDevice *self, guint remove_delay)
 	priv->remove_delay = remove_delay;
 }
 
+/**
+ * fu_device_get_insert_delay:
+ * @self: A #FuDevice
+ *
+ * Returns the delay that should be waited before attempting to access the
+ * inserted device.
+ *
+ * Returns: time in milliseconds
+ *
+ * Since: 1.3.3
+ **/
+guint
+fu_device_get_insert_delay (FuDevice *self)
+{
+	FuDevicePrivate *priv = GET_PRIVATE (self);
+	g_return_val_if_fail (FU_IS_DEVICE (self), 0);
+	return priv->insert_delay;
+}
+
+/**
+ * fu_device_set_insert_delay:
+ * @self: A #FuDevice
+ * @insert_delay: the value in ms
+ *
+ * Sets the delay that should be waited before attempting to access the
+ * inserted device.
+ *
+ * NOTE: This should only be used for devices that fail to respond. If possible
+ * attempt to poll the device in the plugin rather than sleeping a hardcoded
+ * amount of time.
+ *
```

Comment:
Given it's no longer a hardcoded sleep in the daemon, is polling still preferred?